setup.py

```diff
@@ -13,4 +13,4 @@
       include_package_data=True,
       install_requires=requirements,
       description='Happi Database Access for LCLS Beamline Devices',
-      entry_points={'console_scripts': ['happi=happi.cli:main']})
+      entry_points={'console_scripts': ['happi=happi.cli:happi_cli']})
```

Comment:
This rename will force me to finally update the installs in the dev environment

Reply:
I could change it back it's unwanted work, but I didn't have any issues invoking `happi`, so I left that way.

Reply:
Ahh I see, the old entrypoint still exists, so it still works with just a PYTHONPATH update rather than needing a full reinstall

I don't mind either way